alibi/explainers/integrated_gradients.py

```diff
@@ -381,6 +388,7 @@ def _gradients_layer(model: Union[tf.keras.models.Model],
                      orig_dummy_input: Union[list, np.ndarray],
                      x: Union[List[tf.Tensor], tf.Tensor],
                      target: Union[None, tf.Tensor],
+                     forward_kwargs: Optional[dict] = None,
                      compute_layer_inputs_gradients: bool = False) -> tf.Tensor:
```

Comment:
If we're renaming `compute_layer_inputs_gradients` to `attribute_to_layer_inputs` we should do it everywhere in the module, including the private methods.